qa/tasks/cephfs/test_volumes.py

```diff
@@ -668,14 +668,16 @@ def test_volume_info_pending_subvol_deletions(self):
         subvolname = self._generate_random_subvolume_name()
         # create subvolume
         self._fs_cmd("subvolume", "create", self.volname, subvolname, "--mode=777")
-        # create 3K files of 0.1MB
-        self._do_subvolume_io(subvolname, number_of_files=3000, file_size=0.1)
+        # create 3K zero byte files
+        self._do_subvolume_io(subvolname, number_of_files=3000, file_size=0)
```

Comment:
file_size is getting formatted to int before passing to `dd` command. Hence intentionally passing count as 0

Reply:
I didn't understand this o_O

Reply:
@vshankar The `file_size` is passed to `self.mount_a.write_n_mb`  as `n_mb`. See below that `n_mb` is formatted to int which is a count to `dd` cmd. So essentially this interface can't create a file smaller than 1MB

```
  def _do_subvolume_io(self, subvolume, subvolume_group=None, create_dir=None,
                         number_of_files=DEFAULT_NUMBER_OF_FILES, file_size=DEFAULT_FILE_SIZE):
        ...
        ...
        for i in range(number_of_files):
            filename = "{0}.{1}".format(TestVolumes.TEST_FILE_NAME_PREFIX, i)
            self.mount_a.write_n_mb(os.path.join(io_path, filename), file_size)

```
```
    def write_n_mb(self, filename, n_mb, seek=0, wait=True):
        """
        Write the requested number of megabytes to a file
        """
        assert(self.is_mounted())

        return self.run_shell(["dd", "if=/dev/urandom", "of={0}".format(filename),
                               "bs=1M", "conv=fdatasync",
                               "count={0}".format(int(n_mb)),
                               "seek={0}".format(int(seek))
                               ], wait=wait)

```

Reply:
shouldn't we then try create files of 1MB then? in order to make sure everything goes as planned even for files with some data in it?

Reply:
No, not required for this test case. It simple occupies 3GB of memory.